src/App.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+import TodoItem from './TodoItem';
+import TodoList from './TodoList';
+import TodoInput from './TodoInput';
+import { tsAnyKeyword } from '@babel/types';
+
+class App extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      text : '',
+      todoItems : []
+    }
+    this.addTodoItem = this.addTodoItem.bind(this);
+    this.onChange = this.onChange.bind(this);
+    this.deleteTodoItem = this.deleteTodoItem.bind(this);
+    this.changeCompleteTodo = this.changeCompleteTodo.bind(this);
+  }
+  onChange(e){
+    this.setState({
+      text : e.target.value
+    })
+  }
+  addTodoItem(){
+    let text = this.state.text;
+    let id = this.state.todoItems.length;
+    if(text !== ''){
+      this.setState({
+        todoItems : [...this.state.todoItems, 
+          {id: id, text : text, date : new Date, isDone : false}
+        ],
+        text : ''
+      })
+    } else {
+      alert('Incorrect task name')
+    }
+  }
+  deleteTodoItem(index){
+    const items = this.state.todoItems;
+    items.splice(index, 1)
+    this.setState({todoItems : items})
+  }
+  changeCompleteTodo(item){
+    (item.isDone) ? item.isDone = false : item.isDone = true;
```

Comment:
1. Не мутируй аргументы функции. Можешь получить неожиданные баги, связанные с передачей по ссылке.
2. Тернарники можно так использовать.
```suggestion
    item.isDone = item.isDone ? false : true;
```
И логика не очень. если нужно поменять булиан, можно сделать item.isDone = !item.isDone.
Но код этот я не понимаю.